src/main/java/org/apache/maven/plugins/jlink/AbstractJLinkToolchainExecutor.java

```diff
@@ -35,11 +36,11 @@
 
 abstract class AbstractJLinkToolchainExecutor extends AbstractJLinkExecutor
 {
+    private static final Logger LOGGER = LoggerFactory.getLogger( AbstractJLinkToolchainExecutor.class );
```

Comment:
usually "logger" because the object isn't truly a constant even if the variable is

Reply:
This is the first time I heard this. Do you have a reference? I haven't seen any static code analysis tool which can differentiate between `static final mutable` and `static final immutable`.

Reply:
Section 5.2.4 of the Google Java Style Guide explains the thinking here:

https://google.github.io/styleguide/javaguide.html#s5.2-specific-identifier-names

Reply:
@elharo I looked in [other Maven code ](https://github.com/apache/maven-artifact-transfer/blob/dfb1e61c4f5db6fe167b3d879a37ab5e25c8475c/src/main/java/org/apache/maven/shared/transfer/project/deploy/internal/DefaultProjectDeployer.java#L52) that use slf4j logger and there "LOGGER" is used. So I would prefer LOGGER